website/docs/reference/parsing.md

```diff
@@ -51,7 +51,9 @@ If certain inputs change between runs, dbt will trigger a full re-parse. The res
 - `dbt_project.yml` content (or `env_var` values used within)
 - installed packages
 - dbt version
-- certain widely-used macros, e.g. [builtins](/reference/dbt-jinja-functions/builtins) overrides or `generate_x_name` for `database`/`schema`/`alias`
+- certain widely-used macros (for example, [builtins](/reference/dbt-jinja-functions/builtins), overrides, or `generate_x_name` for `database`/`schema`/`alias`)
+
+If you're triggering [CI](/docs/deploy/continuous-integration) job runs, the partial parsing benefits are not available on a new pull request (PR) or new branch. However, they are available on subsequent commits to that new PR or branch. 
```

Comment:
this was added per [this Notion block](https://www.notion.so/dbtlabs/Partial-parse-caching-0fab7d565ae34410ac7c842d7144afd8?pvs=4#5502df0137fc4ec3a290c3cc38762b9a)